libs/btp/tool-header/src/components/tool-header/tool-header.component.html

```diff
@@ -0,0 +1,140 @@
+<div fdbToolHeaderGroup [class.fd-tool-header--menu]="showMenuButton">
+    <div fdbToolHeaderElement *ngIf="fdbToolHeaderState().backButtonVisible">
+        <button fd-button fdbToolHeaderButton glyph="slim-arrow-left" (click)="searchFieldExpanded.set(false)"></button>
+    </div>
+    <div fdbToolHeaderElement *ngIf="fdbToolHeaderState().menuButtonVisible && showMenuButton">
+        <button
+            fd-button
+            fdbToolHeaderButton
+            [title]="'btpToolHeader.menuButtonAriaLabel' | fdTranslate"
+            [ariaLabel]="'btpToolHeader.menuButtonAriaLabel' | fdTranslate"
+            glyph="menu2"
+            (keydown.arrowLeft)="_handleMenuLeftArrowKey()"
+            (keydown.arrowRight)="_handleMenuRightArrowKey()"
+            (click)="menuClick.emit()"
+        ></button>
+    </div>
+    <div fdbToolHeaderElement *ngIf="fdbToolHeaderState().logoVisible">
+        <ng-container *ngIf="_toolHeaderLogoDirective; else defaultLogo">
+            <ng-content select="img[fdbToolHeaderLogo]"></ng-content>
+        </ng-container>
+        <ng-template #defaultLogo>
+            <img
+                fdbToolHeaderLogo
+                (click)="logoClick.emit()"
+                src="https://unpkg.com/fundamental-styles/dist/images/sap-logo.png"
+                srcset="
+                    https://unpkg.com/fundamental-styles/dist/images/sap-logo@2x.png 1x,
+                    https://unpkg.com/fundamental-styles/dist/images/sap-logo@3x.png 2x,
+                    https://unpkg.com/fundamental-styles/dist/images/sap-logo@4x.png 3x
+                "
+                alt="SAP logo"
+                aria-label="SAP logo"
+                title="SAP logo"
```

Comment:
I think this should be customizable and not hard-coded. It will probably require translate, too

Reply:
`img[fdbToolHeaderLogo]` allows to modify entire behavior of that component. That is just a default image

Reply:
But yes, doc would be nice 😆